api/backend_test.go

```diff
@@ -28,6 +28,22 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+var (
+	settings  = accounts.Settings{
```

Comment:
where did these creds come from?  just curious

Reply:
from an account I created with status a few days ago. they are also in the rfc-002